reference/informative_preamble.rst

```diff
@@ -0,0 +1,43 @@
+.. _inform_preamble:
```

Comment:
No way to have some kind of include file? It looks like it overlaps quite a bit (entirely) with `formal.rst`.

Reply:
True, but there are differences in the order of where things will appear, and eventually they will be more distinct as we get more information about the informative spec in the inform_preamble file.  I've found that using includes does odd stuff to the numbering and structure of the PDF, so need to keep that side as is for now.